icepyx/core/read.py

```diff
@@ -54,6 +54,44 @@ def _make_np_datetime(df, keyword):
     return df
 
 
+def _get_track_type_str(grp_path) -> tuple[str, str]:
+    """
+    Determine whether the product contains ground tracks, paths, or profiles and
```

Comment:
The `pt` in `pt1` of ATL11 actually stands for 'pair track'.

```suggestion
    Determine whether the product contains ground tracks, pair tracks, or profiles and
```

Reply:
Thanks for sharing that wisdom!